diff --git a/go.mod b/go.mod
index 1c90cfe4fa..f5ba42a9f3 100644
--- a/go.mod
+++ b/go.mod
@@ -72,7 +72,6 @@ require (
 	golang.org/x/net v0.0.0-20190724013045-ca1201d0de80
 	golang.org/x/oauth2 v0.0.0-20190319182350-c85d3e98c914
 	golang.org/x/sync v0.0.0-20190423024810-112230192c58
-	golang.org/x/xerrors v0.0.0-20190410155217-1f06c39b4373
 	gopkg.in/alexcesaro/quotedprintable.v3 v3.0.0-20150716171945-2caba252f4dc // indirect
 	gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d // indirect
 	gopkg.in/ini.v1 v1.46.0
diff --git a/vendor/golang.org/x/xerrors/adaptor_go1_12.go b/vendor/golang.org/x/xerrors/adaptor.go
similarity index 99%
rename from vendor/golang.org/x/xerrors/adaptor_go1_12.go
rename to vendor/golang.org/x/xerrors/adaptor.go
index 6b9f2f0c7a..4317f24833 100644
--- a/vendor/golang.org/x/xerrors/adaptor_go1_12.go
+++ b/vendor/golang.org/x/xerrors/adaptor.go
@@ -2,8 +2,6 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// +build !go1.13
-
 package xerrors
 
 import (
diff --git a/vendor/golang.org/x/xerrors/adaptor_go1_13.go b/vendor/golang.org/x/xerrors/adaptor_go1_13.go
deleted file mode 100644
index 57f519dc5a..0000000000
--- a/vendor/golang.org/x/xerrors/adaptor_go1_13.go
+++ /dev/null
@@ -1,55 +0,0 @@
-// Copyright 2018 The Go Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style
-// license that can be found in the LICENSE file.
-
-// +build go1.13
-
-package xerrors
-
-import (
-	"errors"
-	"fmt"
-	"strconv"
-)
-
-// A Frame contains part of a call stack.
-type Frame = errors.Frame
-
-// Caller returns a Frame that describes a frame on the caller's stack.
-// The argument skip is the number of frames to skip over.
-// Caller(0) returns the frame for the caller of Caller.
-var Caller func(skip int) Frame = errors.Caller
-
-// FormatError calls the FormatError method of f with an errors.Printer
-// configured according to s and verb, and writes the result to s.
-func FormatError(f Formatter, s fmt.State, verb rune) {
-	// Assuming this function is only called from the Format method, and given
-	// that FormatError takes precedence over Format, it cannot be called from
-	// any package that supports errors.Formatter. It is therefore safe to
-	// disregard that State may be a specific printer implementation and use one
-	// of our choice instead.
-
-	width, okW := s.Width()
-	prec, okP := s.Precision()
-
-	// Construct format string from State s.
-	format := []byte{'%'}
-	if s.Flag('-') {
-		format = append(format, '-')
-	}
-	if s.Flag('+') {
-		format = append(format, '+')
-	}
-	if s.Flag(' ') {
-		format = append(format, ' ')
-	}
-	if okW {
-		format = strconv.AppendInt(format, int64(width), 10)
-	}
-	if okP {
-		format = append(format, '.')
-		format = strconv.AppendInt(format, int64(prec), 10)
-	}
-	format = append(format, string(verb)...)
-	fmt.Fprintf(s, string(format), f)
-}
diff --git a/vendor/golang.org/x/xerrors/format_go1_12.go b/vendor/golang.org/x/xerrors/format.go
similarity index 98%
rename from vendor/golang.org/x/xerrors/format_go1_12.go
rename to vendor/golang.org/x/xerrors/format.go
index ba856feab3..1bc9c26b97 100644
--- a/vendor/golang.org/x/xerrors/format_go1_12.go
+++ b/vendor/golang.org/x/xerrors/format.go
@@ -2,8 +2,6 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// +build !go1.13
-
 package xerrors
 
 // A Formatter formats error messages.
diff --git a/vendor/golang.org/x/xerrors/format_go1_13.go b/vendor/golang.org/x/xerrors/format_go1_13.go
deleted file mode 100644
index 95c65968bc..0000000000
--- a/vendor/golang.org/x/xerrors/format_go1_13.go
+++ /dev/null
@@ -1,19 +0,0 @@
-// Copyright 2018 The Go Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style
-// license that can be found in the LICENSE file.
-
-// +build go1.13
-
-package xerrors
-
-import "errors"
-
-// A Formatter formats error messages.
-type Formatter = errors.Formatter
-
-// A Printer formats error messages.
-//
-// The most common implementation of Printer is the one provided by package fmt
-// during Printf (as of Go 1.13). Localization packages such as golang.org/x/text/message
-// typically provide their own implementations.
-type Printer = errors.Printer
diff --git a/vendor/golang.org/x/xerrors/frame_go1_12.go b/vendor/golang.org/x/xerrors/frame.go
similarity index 98%
rename from vendor/golang.org/x/xerrors/frame_go1_12.go
rename to vendor/golang.org/x/xerrors/frame.go
index 6740f7fa6b..0de628ec50 100644
--- a/vendor/golang.org/x/xerrors/frame_go1_12.go
+++ b/vendor/golang.org/x/xerrors/frame.go
@@ -2,8 +2,6 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// +build !go1.13
-
 package xerrors
 
 import (
